ConsoleApp/Models/Context.cs

```diff
@@ -0,0 +1,39 @@
+namespace ConsoleApp.Models
+{
+    public class Context
+    {
+        private List<Record> Records = new List<Record>();
+        public decimal Balance { get { return Records.Sum(x => x.Summ); } }
+        public List<string> Operations { get { return Records.Select(x => x.ToString()).ToList(); } }
+
+        public void Deposit(decimal summ)
+        {
+            Records.Add(new Record() { Summ = summ, Id = Records.Count });
```

Comment:
deposit/withdraw do not check if value passed negative or positive, I'll be able to pass incorrect values